src/prometheus_alert_rules/memory.rules

```diff
@@ -35,7 +35,12 @@ groups:
           LABELS = {{ $labels }}
         The 5-minute-ahead prediction is made as a linear regression from the last 30 minutes of data.
   - alert: HostMemoryFull
-    expr: avg_over_time(node_memory_MemUsed_percentage[1m]) > 95
+    expr: |
+      100 * avg_over_time(node_memory_MemFree_bytes[1m]) /
+        (
+          avg_over_time(node_memory_MemTotal_bytes[1m])
+          - avg_over_time(node_memory_Hugetlb_bytes[1m])
```

Comment:
Should we add a comment here about diff of averages being better than average of diff? (Less noisy.)

Reply:
Done!